src/cca/scripts/background.ts

```diff
@@ -82,3 +120,19 @@ Promise.all([serversListPagePromise, intents.GetGlobalIntentInterceptor()]).then
     }
   }
 );
+
+// Add some mock servers if we are running as a Chrome extension.
+serversListPagePromise.then((serverListPage) => {
```

Comment:
OK, I did this - now if `localStorage` is unavailable a mock `ServerRepository` is returned. This allows me easily mock unreachable server behaviour and in doing so I discovered a bug in `server_list.ts` - thanks :-)